app/src/main/java/com/sschuraytz/charcoaldrawing/Tool.java

```diff
@@ -126,5 +128,13 @@ public int getRadius()
         return radius;
     }
 
+    public void updateOpacity(int value)
+    {
+        opacity += value;
```

Comment:
A different way to do this would be to put the limit of 255 in here and then return true/false if its outside the range of 0-255

Reply:
> A different way to do this would be to put the limit of 255 in here and then return true/false if its outside the range of 0-255

This could work. But, in this case, since 'lighter' and 'darker' have different checks, but both methods use this opacity method, I don't think it's worth it to combine and have to make two checks for either one action.